primedev/plugins/interfaces/IPluginId.h

```diff
@@ -26,6 +27,10 @@ class IPluginId
 public:
 	virtual const char* GetString(PluginString prop) = 0;
 	virtual int64_t GetField(PluginField prop) = 0;
+
+	static const int64_t COLOR_R_MASK = 0xFF;
```

Comment:
Any reason for this to be in the header when its only needed in the constructor which is in a source file?